src/Tracking/register.jl

```diff
@@ -70,6 +70,46 @@ function align_centroids(im1::AbstractArray{Bool}, im2::AbstractArray{Bool})
     return im1_padded, im2_padded
 end
 
+
+"""
+    shape_difference(floe1::DataFrameRow, floe2::DataFrameRow)
+    
+    Computes the shape difference between two ice floes using the estimated floe
+    orientation to first align both floes on the same axis. 
+
+"""
+function shape_difference(floe1, floe2)
+    im1 = imrotate_bin_nocrop(floe1.mask, floe1.orientation)
+    im2 = imrotate_bin_nocrop(floe2.mask, floe2.orientation)
+    im1, im2 = align_centroids(im1, im2)
+    a_not_b = im1 .> 0 .&& isequal.(im2, 0)
+    b_not_a = im2 .> 0 .&& isequal.(im1, 0)
+    shape_difference = sum(a_not_b .|| b_not_a)
+    return shape_difference
```

Comment:
Deduplicate code by using a call like this?
```suggestion
    shape_difference_ = shape_difference(floe1.mask, floe1.orientation, floe2.mask, floe2.orientation)
    return shape_difference_
```